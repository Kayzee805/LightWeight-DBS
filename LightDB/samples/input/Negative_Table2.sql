SELECT N1.b,N2.b FROM Negatives N1, Negatives N2 WHERE N1.a<0 AND N2.a>-5 AND N1.a>N2.a AND N1.b!=N2.b ORDER BY N1.b;