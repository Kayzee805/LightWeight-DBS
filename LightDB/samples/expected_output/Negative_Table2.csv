-102,103
-101,101
-101,-102
-101,103
101,-102
101,103